src/pages/docs/packaging-applications/create-packages/octopus-cli.md

```diff
@@ -1,79 +1,144 @@
 ```
 
+When you run this command, you'll be prompted for additional configuration options:
+
+```
+? Base Path .
+? Out Folder .
+? Include patterns
+? Verbose Yes
+? Overwrite Yes
+? Author (leave blank to continue)
+```
+
+Here's what each configuration does:
+
+- **Base Path**: Root folder containing the contents to package (defaults to current directory)
+- **Out Folder**: Folder where the package file will be written (defaults to current directory)
+- **Include patterns**: File patterns to include, relative to base path (defaults to "**" for all files)
+- **Verbose**: Enable verbose output during packaging
+- **Overwrite**: Allow overwriting an existing package file with the same ID/version
+- **Author**: Add author metadata to the package (optional)
+
+You can accept the defaults by pressing Enter, or provide your own values. This will create:
+
 > `OctoWeb.1.0.0.nupkg`
 
-You can also change the output directory with the `--outFolder` parameter, and the folder which will be packed with the `--basePath` parameter.
+You can also change the output directory with the `--out-folder` parameter, and the folder which will be packed with the `--base-path` parameter:
 
 ```powershell
-dotnet octo pack --id="OctoWeb" --version="1.0.0" --basePath="folder/to/pack" --outFolder="destination/folder/path"
+octopus package nuget create --id="OctoWeb" --version="1.0.0" --base-path="folder/to/pack" --out-folder="destination/folder/path"
 ```
 
-## Creating Zip packages {#create-zip-packages}
 
-By default, `octo` will create NuGet packages. You can specify Zip packages with the `--format` parameter:
+### Create ZIP packages {#create-zip-packages}
+
+At a minimum, `octopus package zip create` requires an ID:
 
 ```powershell
-dotnet octo pack --id="OctoWeb" --version="1.0.0.0" --format=zip
+octopus package zip create --id="OctoWeb"
 ```
 
->  `OctoWeb.1.0.0.zip`
+The above command will generate a ZIP package in the current working directory with a time-stamp based version number such as:
+
+> `OctoWeb.2018.6.26.190140.zip`
+
+If you want to provide your own version, you can pass the `--version` parameter:
+
+```powershell
+octopus package zip create --id="OctoWeb" --version="1.0.0"
+```
+
+When you run this command, you'll be prompted for additional configuration options:
+
+```
+? Base Path .
+? Out Folder .
+? Include patterns
+? Verbose Yes
+? Overwrite Yes
+? Author (leave blank to continue)
+```
+
+Here's what each configuration does:
+
+- **Base Path**: Root folder containing the contents to zip (defaults to current directory)
+- **Out Folder**: Folder where the zip file will be written (defaults to current directory)
+- **Include patterns**: File patterns to include, relative to base path (defaults to "**" for all files)
+- **Verbose**: Enable verbose output during packaging
+- **Overwrite**: Allow overwriting an existing package file with the same ID/version
+- **Author**: Add author metadata to the package (optional)
```

Comment:
Having this duplicated for nuget and zip feels a bit redundant to me, do you think we could simplify this page to reduce that duplication?